pkg/controller/accountclaim/accountclaim_controller.go

```diff
@@ -263,6 +263,16 @@ func (r *ReconcileAccountClaim) Reconcile(request reconcile.Request) (reconcile.
 func (r *ReconcileAccountClaim) setSupportRoleARNManagedOpenshift(reqLogger logr.Logger, accountClaim *awsv1alpha1.AccountClaim, account *awsv1alpha1.Account) error {
 	if accountClaim.Spec.STSRoleARN == "" {
 		instanceID := account.Labels[awsv1alpha1.IAMUserIDLabel]
+
+		// get configMap and check if account is fedramp
+		configMap, err := controllerutils.GetOperatorConfigMap(r.client)
+		if err != nil {
+			log.Error(err, "failed retrieving configmap")
+		}
+		_, ok := configMap.Data["fedramp"]
+		if ok {
```

Comment:
Something about getting the configmap every time we need to check this here feels wrong to me, especially for a var that really only needs to be read once at startup and then never again.

Maybe this should go into the `newReconciler` function (or whatever it's called, I can't remember off the top of my head) and we set a new struct property on the Reconciler that is `Fedramp bool` which then allows us to do things like

```golang
accountClaim.Spec.SupportRoleARN = fmt.Sprintf....

if r.FedRamp {
    accountClaim.Spec.SupportRoleARN = fmt.Sprintf....
}
```

Reply:
Alternatives are to get it once during operator startup and make it globally available, since the operator will only be running on a fedramp hive or a regular hive, OR getting it once at the start of the reconcile. I prefer the former tbh since its a global config for the whole operator

Reply:
changes pushed. now using the newReconciler func per suggestion!